packages/components/pie-avatar/src/index.ts

```diff
@@ -114,7 +129,7 @@ export class PieAvatar extends RtlMixin(PieElement) implements AvatarProps {
             return html`<a data-test-id="pie-avatar-anchor">${content}</a>`;
         }
 
-        return html`<div class="c-avatar-content" data-test-id="pie-avatar-div">${content}</div>`;
+        return html`<div class="${!this.src ? 'c-avatar-content' : ''}" data-test-id="pie-avatar-div" ?aria-hidden="${this.src}">${content}</div>`;
```

Comment:
related to my comment in the scss file, you could apply `c-avatar` to the outermost element. So the `div`, `button` and `a` tags.